src/reviews.js

```diff
@@ -0,0 +1,161 @@
+const axios = require('axios')
+const cheerio = require('cheerio')
+const { getResultsFromSearch } = require('./info.js')
+
+const BASE_URI = 'https://myanimelist.net/anime/'
+const NUMBER_REVIEWS_BY_PAGE = 20
+const INITIAL_FIRST_PAGE_REVIEW = 1
+
+/**
+* Return a formatter javascript date
+* @params malDate string a date in a string object
+* @return date The string parse to a date
+**/
+const malDateToJsDate = (malDate) => {
+  return new Date(malDate)
+}
+
+/**
+* Return a formatted javascript number
+* @params malNumber string a number in a string object
+* @return number The string parse to a number
+**/
+const malNumberToJsNumber = (malNumber) => {
+  return malNumber ? Number(malNumber) : 0
+}
+
+const parsePage = ($) => {
+  const items = $('.borderDark')
+  const result = []
+
+  items.each(function (elem) {
+    const notes = $(this).find('.spaceit.pt8 div')
+    const reviewMore = $(this).find('.spaceit.pt8 span')
+    // For presenting the review only without the notes
+    $(this).find('.spaceit.pt8 div').remove()
+    $(this).find('.spaceit.pt8 span').remove()
+    $(this).find('.spaceit.pt8 a.js-toggle-review-button').remove()
+
+    result.push({
+      author: $($(this).find('.spaceit td:nth-child(2) a')['0']).text().trim(),
+      date: malDateToJsDate($($(this).find('.spaceit .mb8 div')['0']).text().trim()),
+      seen: $(this).find('.spaceit .mb8 .lightLink').text().trim(),
+      note_overall: malNumberToJsNumber($(notes).find('tr:nth-child(1) td:nth-child(2)').text().trim()),
+      note_story: malNumberToJsNumber($(notes).find('tr:nth-child(2) td:nth-child(2)').text().trim()),
+      note_animation: malNumberToJsNumber($(notes).find('tr:nth-child(3) td:nth-child(2)').text().trim()),
+      note_sound: malNumberToJsNumber($(notes).find('tr:nth-child(4) td:nth-child(2)').text().trim()),
+      note_character: malNumberToJsNumber($(notes).find('tr:nth-child(5) td:nth-child(2)').text().trim()),
+      note_enjoyment: malNumberToJsNumber($(notes).find('tr:nth-child(6) td:nth-child(2)').text().trim()),
```

Comment:
The rest of the library uses cameCase. Would it be ok to change the snake_case to CamelCase?

Also, I think that something like that would be kinda easier to use:
```javascript
note: {
  overall: malNumberToJsNumber($(notes).find('tr:nth-child(1) td:nth-child(2)').text().trim()),
  story: malNumberToJsNumber($(notes).find('tr:nth-child(2) td:nth-child(2)').text().trim()),
  ...
}
```

Reply:
Ha yes, I did not pay attention to that.
I made the change and remove the `note_`
I also change the package json to all tests :  *.tests